store-consumer/src/main/java/com/stormeye/event/store/service/storage/impl/block/BlockAddedService.java

```diff
@@ -67,18 +81,17 @@ public Block store(final EventInfo eventInfo) {
     }
 
     private boolean isDuplicateEventException(Exception e) {
-        return e instanceof DataIntegrityViolationException && e.getMessage().contains("UKIDXE_EVENT_ID_BLOCK_HASH_INDEX");
+        return e instanceof DataIntegrityViolationException && e.getMessage().contains("UKIDX_EVENT_ID_BLOCK_HASH");
     }
 
     @NotNull
+    private Block storeBlock(final EventInfo eventInfo) throws NoSuchAlgorithmException {
 
+        final BlockAdded toStore = (BlockAdded) eventInfo.getData();
 
-    private Block storeBlock(final Long eventId, final BlockAdded toStore) {
-        //  final BlockAdded toStore = (BlockAdded) eventInfo.getData();
-
-        final Block block;
+        logger.info("Processing BlockAdded event. BlockHash: {} ", toStore.getBlockHash());
 
-        block = this.blockRepository.save(
+        final Block block = this.blockRepository.save(
```

Comment:
The @Builder annotation in Lombok is useful
Use it to create large objects, you can then see at a glance what you're assigning to what